### PR TITLE
Add disk space check to krel stage/release

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
The prerequisites checker now features the missing disk space check,
which is right now hard-coded to 100GB of available space.

#### Which issue(s) this PR fixes:

Refers to #1673 
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/release/prerequisites.go [hunk 1]
```diff
@@ -80,10 +83,26 @@ func (*defaultPrerequisitesChecker) DockerVersion() (string, error) {
 	return res.OutputTrimNL(), err
 }
 
-func (p *PrerequisitesChecker) Run() error {
+func (*defaultPrerequisitesChecker) DiskSpaceAvailable(
```
Comment (cpanato) [inline pkg/release/prerequisites.go:86]: why we dont use something like: https://github.com/shirou/gopsutil/

Comment (saschagrunert) [reply pkg/release/prerequisites.go]: Makes sense, changed the implementation to use the new dependency. :+1:

File: pkg/release/prerequisites.go [hunk 2]
```diff
@@ -80,10 +82,14 @@ func (*defaultPrerequisitesChecker) DockerVersion() (string, error) {
...
-func (p *PrerequisitesChecker) Run() error {
+func (*defaultPrerequisitesChecker) Usage(dir string) (*disk.UsageStat, error) {
+	return disk.Usage(dir)
+}
+
+func (p *PrerequisitesChecker) Run(workdir string) error {
 	// Command checks
 	commands := []string{
-		"docker", "jq", "gsutil", "gcloud", "ssh",
+		"docker", "jq", "gsutil", "gcloud", "ssh", "sed",
```
Comment (cpanato) [inline pkg/release/prerequisites.go:92]: do we still need `sed`

File: pkg/release/prerequisites.go (hunk 1 above)
Comment (cpanato) [reply pkg/release/prerequisites.go]: cool

File: pkg/release/prerequisites.go (hunk 2 above)
Comment (saschagrunert) [reply pkg/release/prerequisites.go]: No, removed it again.

**Review by cpanato — APPROVED** [review]
this is cool
thanks for adding that

/lgtm

**Pull request merged**